packages/botonic-core/src/hubtype-service.js

```diff
@@ -27,7 +28,7 @@ export class HubtypeService {
         headers: {
           'X-BOTONIC-USER-ID': this.user.id,
```

Comment:
since I had a hard time finding these headers in backend:
 
I suggest:
*adding a comment here like
// See https://stackoverflow.com/a/22856867/145289 on why the headers use dashes here but underscore in backend
* asking Marcos why in backend he checked both "HTTP-X-BOTONIC" and "X-BOTONIC" https://github.com/metis-ai/hubtype-backend/blob/develop/organization/views/provider_account.py#L525